.github/workflows/tests.yml

```
@@ -0,0 +1,47 @@
+name: Tests
+on: [push, pull_request]
+
+jobs:
+  test-ubuntu-ruby:
+    runs-on: ubuntu-latest
+    services:
+      postgis:
+        image: postgis/postgis:${{matrix.pg}}
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --name=postgres
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    env:
+      BUNDLE_GEMFILE: ${{matrix.gemfile}}
+    strategy:
+      matrix:
+        ruby: [3.0, 2.7, 2.6, 2.5, jruby]
```

```suggestion
        ruby: ['3.0', '2.7', '2.6', '2.5', jruby]
```

lets follow the rules we've applied in rgeo/rgeo !